assets/js/base/components/cart-checkout/totals/totals-fees-item/index.js

```diff
@@ -27,6 +27,7 @@ const TotalsFeesItem = ( { currency, values } ) => {
 
 	return (
 		<TotalsItem
+			className="wc-block-components-totals-fees"
```

Comment:
Yeah, that's the idea, all others have a class name, so I thought it would make sense adding a class name to this one too so themes had a way to style it.